tests/fragments/google/api/field_behavior.proto

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 Google LLC
```

Comment:
Isn't this file already in common protos?

Reply:
Yes it is, but I just copied it in here instead of trying to clone the repo as part of fragment testing.